### DISCUSSION TITLE
Github student

### DISCUSSION THREAD

**Discussion by Gonzalodelafuenteher** [Genuitec/Copilot4Eclipse-community#63] [Help]
Hello! I need help to put copilot in the Eclipse IDE. I have the github student approved and supposedly they have already given me the benefits, but when I link my account with the copilot eclipse extension it tells me that I do not have that benefit or that it has already expired

**Comment by wayneparrott** [comment]
You should be able to determine the status of your GitHub Copilot subscription from the GitHub dashboard: https://github.com/settings/copilot
